src/qt/networkstyle.cpp

```diff
@@ -20,78 +23,72 @@ static const struct {
     {"test", QAPP_APP_NAME_TESTNET, 70, 30, QT_TRANSLATE_NOOP("SplashScreen", "[testnet]")},
     {"regtest", QAPP_APP_NAME_TESTNET, 160, 30, "[regtest]"}
 };
-static const unsigned network_styles_count = sizeof(network_styles)/sizeof(*network_styles);
+static const int network_styles_count = sizeof(network_styles)/sizeof(*network_styles);
```

Comment:
why unsigned? Its not needed and almost all Qt apis avoid unsigned, so it would be inconsistent.